meross_iot/cloud/device_factory.py

```diff
@@ -32,5 +32,7 @@ def build_subdevice_wrapper(cloud_client,
                             ):
     if device_type.startswith('mts'):
         return ValveSubDevice(cloud_client, subdevice_id=device_id, parent_hub=parent_hub, **device_specs)
+    elif device_type.startswith('ms100'):
```

Comment:
Instead of checking against "ms100", just check against "msXXX" where XXX are digits. Let's use regex here..

Reply:
done